workflow/cron/operator.go

```diff
@@ -98,7 +98,6 @@ func (woc *cronWfOperationCtx) validateCronWorkflow() error {
 		woc.reportCronWorkflowError(v1alpha1.ConditionTypeSpecError, fmt.Sprint(err))
 	} else {
 		woc.cronWf.Status.Conditions.RemoveCondition(v1alpha1.ConditionTypeSpecError)
-		woc.persistUpdate()
```

Comment:
this is the bug I think, cannot reapplyUpdates twice

Reply:
Good catch